core/src/main/java/feast/core/job/dataflow/DataflowJobManager.java

```diff
@@ -151,6 +154,19 @@ private ImportOptions getPipelineOptions(String jobName, List<FeatureSetSpec> fe
     pipelineOptions.setUpdate(update);
     pipelineOptions.setRunner(DataflowRunner.class);
     pipelineOptions.setJobName(jobName);
+
+    // "DataflowRunner.detectClassPathResourcesToStage" runs into this issue when trying to detect jars
+    // to upload, when Feast Core starts from a packaged jar.
+    //
+    // https://stackoverflow.com/questions/48961440/illegalargumentexception-unable-to-convert-url-jarfile-app-jar-boot-inf-cla
+    // https://stackoverflow.com/questions/48292491/java-dataflow-unable-to-use-classloader-to-detect-classpath-elements
+    //
+    // Set the files to stage manually, following the quick workaround in Stackoverflow.
+    // Perhaps there is a cleaner solution?
```

Comment:
I don't feel like this final question adds any value

Reply:
No longer using this method because it wasn't working.